src/main/java/Deadlines.java

```diff
@@ -0,0 +1,23 @@
+public class Deadlines extends Task {
```

Comment:
Perhaps the class name could be singular? 
I like that you used inheritance for different types of Task.